mmaction/models/losses/cross_entropy_loss.py

```diff
@@ -9,6 +9,20 @@
 class CrossEntropyLoss(BaseWeightedLoss):
     """Cross Entropy Loss.
 
+    Support two kinds of labels and their corresponding loss type. It's worth
+    mentioning that loss type will be detected by the shape of ``cls_score``
+    and ``label``.
+    1) Sparse label: This label is an integer array and all of the elements are
```

Comment:
Sparse -> One-hot

Reply:
Non-sparse labels are one-hot like labels, aka a probability distribution.

Sparse label is a list of integers.

Reply:
I think this is a misuse of "sparse". How about just "labels"

Reply:
I'm not sure about the misuse thing.

[TF SparseCategoricalCrossentropy](https://www.tensorflow.org/api_docs/python/tf/keras/losses/SparseCategoricalCrossentropy)

[Book: Hands-On Machine Learning book with Scikit-Learn and TF](https://stats.stackexchange.com/questions/427748/what-does-it-mean-when-someone-says-they-have-sparse-labels)

[GluonCV](https://github.com/dmlc/gluon-cv/blob/2da042f8982b8c46cdf04b0646c845f1175402ef/gluoncv/loss.py#L275)

Reply:
It seems that the usage comes from tf. It is not good term, lets stop its popularization.

Okay joking :) I think "label" is clear enough here. If you want to differentiate, use "hard label"